CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ library-definition test runner.
 ***We call this the "flow version directory".***
 
 This specifies that the definition you are contributing is compatible with the
-version range of the directiry. You MUST specify a version range with names like
+version range of the directory. You MUST specify a version range with names like
```

Comment:
Nit, but would you mind moving this change do a separate commit, or better another PR?

Reply:
👍